src/ga4gh/vrs/extras/localizer.py

```diff
@@ -45,27 +41,24 @@ def __init__(self):
         self._ana_maps = {k: make_name_ac_map(k) for k in assy_name_to_map_name}
 
 
-    def localize_allele(self, allele):
-        # copy input variant and replace location
-        # N.B. deepcopy leads to recursion errors
+    def localize_allele(self, allele, assembly_name = "GRCh38"):
+        """copy input variant and replace location N.B. deepcopy leads to recursion
+        errors"""
         allele_sl = ga4gh.vrs.models.Variation(**allele.as_dict())
         del allele_sl._id
-        allele_sl.location = self.localize(allele.location)
+        allele_sl.location = self.localize_named_feature(allele.location, assembly_name)
```

Comment:
This doesn't seem right to me by the names and descriptions of these methods. Why would updating a sequence location (parent method) have anything to do with a named feature (child method)? My understanding of named features are annotated regions on a sequence, e.g. gene symbols, protein domains, named exons.

I would need to dig deeper into the code to understand if this is actually correct; and if so we need to reconsider the names and descriptions of these functions.

Reply:
From my understanding of this and the [notebook](https://github.com/ga4gh/vrs-python/blob/issue-97/notebooks/Feature%20Based%20Locations.ipynb), these methods converts a Chromsome Location or Gene Location to a Sequence Location

Reply:
@reece and @ahwagner can you review the changes to the localizer module?

Reply:
@korikuzma is correct: the "localization" process is intended to convert a named feature to coordinates. I proposed (and I thought that you and Larry agreed) that we needed to be able to project all features onto a common coordinate system. This operation is necessary in order to identify overlapping features, for example.

Reply:
Ah, I think I misunderstood; I thought we were "localizing" a sequence location here, but understand now the idea is to localize a feature or chromosome location. That seems appropriately named to me. Thanks @korikuzma and @reece.